src/oc/web/components/stream_view_item.cljs

```diff
@@ -65,7 +65,8 @@
                               :sorted-comments)
                           (:comments activity-data))
         activity-attachments (:attachments activity-data)
-        comment-link (utils/link-for (:links activity-data) "comments")]
+        comments-link (utils/link-for (:links activity-data) "comments")
```

Comment:
Both of these components (stream view item and fullscreen post) are dependent on know something about the comment links. This should be moved to the store and have it be an attribute of the activity/post. `(:can-comment post)` or `(:has-comments post)`